snapshot/lib/snapshot/test_command_generator.rb

```diff
@@ -58,11 +58,11 @@ def verify_devices_share_os(devices)
         return devices.count == 1
       end
 
-      def xcodebuild_log_path(device_type: nil, language: nil, locale: nil)
+      def xcodebuild_log_path(devices: nil, language: nil, locale: nil)
         name_components = [Snapshot.project.app_name, Snapshot.config[:scheme]]
```

Comment:
Currently this is working, but the second execution (if there are more simulators than cores on the machine) will overwrite the first execution's logs. Thinking maybe we should concatenate these, but because of the `:namespace_log_files` param that is checked below, that throws a bit of a wrench into that.